_includes/manuals/1.12/3.2.3_installation_scenarios.md

```diff
@@ -1,6 +1,26 @@
 
 The Foreman installer can accommodate more complex, multi-host setups when supplied with appropriate parameters.
 
+#### Using an external database server
+
+Per default foreman-installer will install a PostgreSQL database server onto the Foreman host and create its database. An external MySQL or PostgreSQL database server with an already created database can be used with the following arguments:
+
+{% highlight bash %}
+foreman-installer \
+  --foreman-db-type=mysql \
+  --foreman-db-manage=false \
+  --foreman-db-host=dbserver.example.com \
+  --foreman-db-database=dbname \
+  --foreman-db-username=dbuser \
+  --foreman-db-password=dbpassword
+{% endhighlight %}
+
+As a post-installation step, to populate the database correctly, run:
+
+{% highlight bash %}
+foreman-rake db:migrate db:seed apipie:cache:index
```

Comment:
Please split the db:migrate/seed into separate commands, we generally don't test them being run in the same process (db:seed + apipie:cache:index should be fine) as db:migrate makes a lot of in-process changes.